src/components/ReportsPage/HeatmapPage/HeatmapPage.tsx

```diff
@@ -86,6 +101,7 @@ export const HeatmapPage = () => {
     const selectedScorecard = scorecardsResult.value?.find((scorecard) => scorecard.id === filters.selectedScorecardId);
 
     const excludedGroupBys = isScorecardTeamBased(selectedScorecard) ? [GroupByOption.TEAM] : [];
+    if (selectedScorecard?.filter?.type === FilterType.DOMAIN_FILTER) excludedGroupBys.push(GroupByOption.DOMAIN);
```

Comment:
Hmm I seem to be able to group by domain when the Scorecard is a domain-based Scorecard in brain-app -- is this correct?